app/views/add-reduction.html

```diff
@@ -16,19 +20,20 @@
   <p class="lede">Add a new reduction for <span class="bold">{{ title }}</span></p>
```

Comment:
You need to include different title when showing reduction vs adding a reduction. Similar to the if else for heading.

Reply:
Add all variable to a single if at top of file